alecs-theme/src/css/partials/_buttons.scss

```diff
@@ -0,0 +1,20 @@
+@import "bourbon";
+@import "mixins";
+
+.btn-theme-default {
+	border-radius: 0;
+	border-width: 0;
+	color: $brand-purple;
+	font-family: 'open_sanssemibold', sans-serif;
+	font-size: 12px;
+	min-width: 262px;
+	outline-width: 0;
+	padding: 13px 20px;
+
+	&:hover, &:focus, &:active {
```

Comment:
We need to update this to follow the psuedo CSS selector rule (https://css-tricks.com/remember-selectors-with-love-and-hate/).